services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java

```diff
@@ -55,6 +55,8 @@ public class KeycloakErrorHandler implements ExceptionMapper<Throwable> {
     public static final String UNCAUGHT_SERVER_ERROR_TEXT = "Uncaught server error";
     public static final String ERROR_RESPONSE_TEXT = "Error response {0}";
 
+    private static final String FAILED_DECODE_URL="Failed to decode URL";
```

Comment:
Could you please add the comment above this with something like:
```
// This error being used by org.jboss.resteasy.reactive.common.util.URLUtils
```

It can be useful in the future just for tracking that this is the error, which is not thrown from our own codebase (and hence we don't have much better ways  to deal with it than what you did in the PR).